person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :id, :name, :age, :parent_permission
+
+  def initialize(id, age)
+    @id = id
+    @name = 'Unknown'
+    @age = age
+    @parent_permission = true
+  end
```

Comment:
- [ ] [REQUIRED] Good job so far 👍 I appreciate what you have done but in this `def` it will be clean if you do something like the method below 😃:

```ruby
...
  def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end
...
```